source/css/_schemes/Pisces/_sidebar.styl

```diff
@@ -120,9 +116,7 @@
 
   .fa {
     margin-right: 2px;
-    font-size: 16px;
   }
-  .fa-globe { font-size: 15px; }
```

Comment:
Screens: before and after?

Reply:
No changes are made. Duplicate definition.

Reply:
Duplicate definition from where?

Reply:
from .fa{font-size:1em}, and .fa-globe is also .fa